pkg/operator/ceph/object/spec.go

```diff
@@ -155,6 +155,24 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	if c.store.Spec.RgwOpsLogEnabled != nil {
+		// If the rgw ops logs are enabled we add the side-car container named rgw-ops-log
+		if *c.store.Spec.RgwOpsLogEnabled {
+			monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
```

Comment:
To simplify the unit tests, we try to only generate the pod specs and nothing else inside these pod spec methods. So the monstore config could be likely applied in the method [setFlagsMonConfigStore()](https://github.com/rook/rook/blob/4c87cf9723fe3522833934d92297a97b51caad95/pkg/operator/ceph/object/config.go#L155).